server/src/services/youtube-transcript.ts

```diff
@@ -496,7 +496,7 @@ export const processYoutubeTranscript = async (
     
     // Include video title in documentName for better readability
     // But keep a consistent id format for duplicate checking
-    const documentName = videoDetails.title;
```

Comment:
no, why you can make a very disruption and incompatibility breaking change like this?